tidoop-mr-lib-api/resources/mysql_db_and_tables.sql

```diff
@@ -0,0 +1,5 @@
+DROP DATABASE cosmos_gui;
+CREATE DATABASE cosmos_gui;
+USE cosmos_gui;
+
+CREATE TABLE tidoop_job (jobId VARCHAR(24) NOT NULL PRIMARY KEY UNIQUE, jobType TEXT NOT NULL, startTime TIMESTAMP DEFAULT '0000-00-00 00:00:00', endTime TIMESTAMP DEFAULT '0000-00-00 00:00:00', mapProgress INT NOT NULL DEFAULT '0', reduceProgress INT NOT NULL DEFAULT '0');
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-tidoop/commit/6a0bdc08d70a95d05f2982f185aad4e716939f5f